qdrant_client/qdrant_fastembed.py

```diff
@@ -137,15 +143,65 @@ def _scored_points_to_query_responses(
             )
         return response
 
+    def _records_iterator(
+        self,
+        ids: Optional[Iterable[models.ExtendedPointId]],
+        metadata: Optional[Iterable[Dict[str, Any]]],
+        encoded_docs: Iterable[Tuple[str, List[float]]],
+        ids_accumulator: list,
+    ) -> Iterable[models.Record]:
+        if ids is None:
+            ids = iter(lambda: uuid.uuid4().hex, None)
+
+        if metadata is None:
+            metadata = iter(lambda: {}, None)
+
+        vector_name = self.get_vector_field_name()
+
+        for idx, meta, (doc, vector) in zip(ids, metadata, encoded_docs):
+            ids_accumulator.append(idx)
+            payload = {"document": doc, **meta}
+            yield models.Record(id=idx, payload=payload, vector={vector_name: vector})
+
+    def get_fastembed_vector_params(
+        self,
+        on_disk: Optional[bool] = None,
+        quantization_config: Optional[models.QuantizationConfig] = None,
+        hnsw_config: Optional[models.HnswConfigDiff] = None,
+    ) -> Dict[str, models.VectorParams]:
+        """
+        Generates vector configuration, compatible with fastembed models.
+
+        Args:
+            on_disk: if True, vectors will be stored on disk. If None, default value will be used.
+            quantization_config: Quantization configuration. If None, quantization will be disabled.
+            hnsw_config: HNSW configuration. If None, default configuration will be used.
+
+        Returns:
+            Configuration for `vectors_config` argument in `create_collection` method.
+        """
+        vector_field_name = self.get_vector_field_name()
+        embeddings_size, distance = self._get_model_params(model_name=self.embedding_model_name)
+        return {
+            vector_field_name: models.VectorParams(
+                size=embeddings_size,
+                distance=distance,
+                on_disk=on_disk,
+                quantization_config=quantization_config,
+                hnsw_config=hnsw_config,
+            )
+        }
+
     def add(
```

Comment:
return type is not updated in the docstring

Reply:
check